spec/integration/arrtest_spec.rb

```diff
@@ -0,0 +1,44 @@
+describe "Array Plot Example" do
+  let(:file_path)    { 'spec/tmp/array_plot.eps' }
+  let(:fixture_path) { 'spec/fixtures/plots/array_plot.eps' }
+
```

Comment:
the idea of this test is rely on a pregenerated file when testing the whole integration